release-notes/1.0/1.0.19189.1-Preview.md

```diff
@@ -0,0 +1,159 @@
+* [UTF-8 support](#utf-8-support) - Available in Microsoft SQL Server SQL Server 2019 since CTP 2.3.
+
+New features over .NET Core 2.2 System.Data.SqlClient.
+
+* [Data Classification](#data-classification) - Available in Azure SQL Database and Microsoft SQL Server 2019 since CTP 2.0.
+* [UTF-8 support](#utf-8-support) - Available in Microsoft SQL Server SQL Server 2019 since CTP 2.3.
+* [Always Encrypted with Enclaves](#always-encrypted-with-enclaves) - Always Encrypted is available in Microsoft SQL Server 2016 and higher. Enclave support was introduced in Microsoft Sql Server 2019 CTP 2.0.
+
+### Known Issues
+
+* User Data Types (UDTs) do not work with Microsoft.Data.SqlClient.
+
+### Data Classification
+
+Data Classification brings a new set of APIs exposing read-only Data Sensitivity and Classification information about objects retrieved via SqlDataReader when the underlying source supports the feature and contains metadata about [data sensitivity and classification](https://docs.microsoft.com/en-us/sql/relational-databases/security/sql-data-discovery-and-classification?view=sql-server-2017):
+
+```C#
+public class SqlDataReader
+{
+    public Microsoft.Data.SqlClient.DataClassification.SensitivityClassification SensitivityClassification
+}
+
+namespace Microsoft.Data.SqlClient.DataClassification
+{
+    public class ColumnSensitivity
+    {
+        public System.Collections.ObjectModel.ReadOnlyCollection<Microsoft.Data.SqlClient.DataClassification.SensitivityProperty> SensitivityProperties
+    }
+    public class InformationType
+    {
+        public string Id
+        public string Name
+    }
+    public class Label
+    {
+        public string Id
+        public string Name
+    }
+    public class SensitivityClassification
+    {
+        public System.Collections.ObjectModel.ReadOnlyCollection<Microsoft.Data.SqlClient.DataClassification.ColumnSensitivity> ColumnSensitivities
+        public System.Collections.ObjectModel.ReadOnlyCollection<Microsoft.Data.SqlClient.DataClassification.InformationType> InformationTypes
+        public System.Collections.ObjectModel.ReadOnlyCollection<Microsoft.Data.SqlClient.DataClassification.Label> Labels
+    }
+    public class SensitivityProperty
+    {
+        public Microsoft.Data.SqlClient.DataClassification.InformationType InformationType
+        public Microsoft.Data.SqlClient.DataClassification.Label Label
+    }
+}
+```
+
+### UTF-8 Support
+
+UTF-8 support does not require any application code changes. These SqlClient changes simply optimize the communication between the client and server when the server supports UTF-8 and the underlying column collation is UTF-8. See the UTF-8 section under [What's new in SQL Server 2019 preview](https://docs.microsoft.com/en-us/sql/sql-server/what-s-new-in-sql-server-ver15?view=sqlallproducts-allversions#utf-8-support-ctp-23).
```

Comment:
I think for the link here and the one below, you want to remove /en-us/ from the URLs.